src/commands/workspace/link.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
Same for `bolt workspace [package] link`

Reply:
Is helpful during migration.  
If packages in other repo are dependent on packages bolt repo.